tools/common_lib/src/dnnl_utils.h

```diff
@@ -84,6 +85,18 @@ inline dnnl::memory::desc to_dnnl_mem_desc(const TensorShape& shape, const DataL
     return dnnl::memory::desc{ to_dnnl_dims(shape), to_dnnl_data_type(t), to_dnnl_format(l) };
 }
 
+inline dnnl::memory::desc gemm_matrix_transpose(const dnnl::memory::desc& nchw_desc, const DataType& t)
```

Comment:
This function is used only in c-tor of umd dispatcher in gemm.h - so does it really have to be added in util header?
I recommend moving it to c-tor of umd dispatcher and making it lambda